src/constants/constants.ts

```diff
@@ -13,6 +13,8 @@ export const connectionApplicationName = 'vscode-mssql';
 export const outputChannelName = 'MSSQL';
 export const connectionConfigFilename = 'settings.json';
 export const connectionsArrayName = 'connections';
+export const disconnectedServerLabel = 'disconnectedServer';
```

Comment:
>disconnectedServerLabel [](start = 13, length = 23)

These aren't really labels (label to me would imply displayed text). NodeType seems more appropriate here